ts/session/types/PubKey.ts

```diff
@@ -54,7 +54,7 @@ export class PubKey {
   public isEqual(comparator: PubKey | string) {
     return comparator instanceof PubKey
       ? this.key === comparator.key
-      : this.key === comparator;
+      : this.key === comparator.toLowerCase();
```

Comment:
When would pubkeys ever include uppercase? Since pubkeys represent a string of char codes, `'A' =/= 'a'`

Reply:
Hence we should consider a pubkey with any uppercase as broadly invalid. We should edit the regex to `a-f` rather than `a-fA-F`

Reply:
This just handles those cases where a pubkey has uppercased letters. A key is case insensitive so it's valid both ways.